requirements.txt

```diff
@@ -1,6 +1,6 @@
 kiwixstorage>=0.8.1,<1.0
 pif>=0.8.2,<0.9
-zimscraperlib>=2.1,<3.0
+zimscraperlib>=3.2.0,<4.0
```

Comment:
This is probably wrong, you need 3.3.0 to have https://github.com/openzim/python-scraperlib/pull/126